includes/admin/class-wp-job-manager-admin.php

```diff
@@ -113,6 +113,11 @@ public function admin_enqueue_scripts() {
 			WP_Job_Manager::register_script( 'job_manager_admin_js', 'js/admin.js', [ 'jquery', 'jquery-tiptip', 'select2' ], true );
 			wp_enqueue_script( 'job_manager_admin_js' );
 
+			WP_Job_Manager::register_script( 'job_manager_setup_js', 'js/admin/job-tags-upsell.js', [], true );
+			if ( ! class_exists( 'WP_Job_Manager_Job_Tags' ) ) {
+				wp_enqueue_script( 'job_manager_setup_js' );
+			}
```

Comment:
WDYT about renaming `job_manager_setup_js` to `job_tags_upsell_js`?

```suggestion
			WP_Job_Manager::register_script( 'job_tags_upsell_js', 'js/admin/job-tags-upsell.js', [], true );
			if ( ! class_exists( 'WP_Job_Manager_Job_Tags' ) ) {
				wp_enqueue_script( 'job_tags_upsell_js' );
			}
```